acquia_lift_profiles/acquia_lift_profiles.admin.inc

```diff
@@ -165,7 +158,7 @@ function acquia_lift_profiles_admin_form($form, &$form_state) {
         if ($i == $min) {
           $form['acquia_lift_profiles_udf_mappings'][$type][$udf]['#prefix'] = '<div class="acquia-lift-profiles-hidden-udfs">';
         }
-        elseif ($i == $max - 1) {
+        if ($i == $max - 1) {
```

Comment:
Changing this "elseif" to "if" is not related to the story. The "elseif" is logically incorrect and introduces misconfiguration at the edge case of $min is 3 and $max is 4. So it is fixed on the side.